assets/styles/_base.scss

```diff
@@ -716,3 +716,39 @@ select {
     }
   }
 }
+
+/**
+* 4.0 Content
+*/
+
+.flyntComponent {
```

Comment:
Please move the whole thing to the upper section _2.0 Layout_ and remove the wrapping .flyntComponent class. I understand that you would like to regroup elements, but lets do that in one go for all existing and upcoming helper classes at a later point in time.